### PR TITLE
[#77] - Command to close Coaster on closing date

### PR DESCRIPTION
**Pull request by richardcrete**

### CODE REVIEW

File: src/Command/CoasterCloseCommand.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $today = new DateTime();
+
+        $closingCoasters = $this->coasterRepository->findBy([
+            'closingDate' => $today,
+        ]);
+
+        $closingStatus = $this->statusRepository->findOneBy(['name' => Status::CLOSED_DEFINITELY]);
```
Comment (florian987) [inline src/Command/CoasterCloseCommand.php:46]: will all scheduled closings correspond to this status?

File: src/Command/CoasterCloseCommand.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Notifier\ChatterInterface;
+use Symfony\Component\Notifier\Message\ChatMessage;
+
+class CoasterCloseCommand extends Command
+{
+    protected static $defaultName = 'coaster:close';
+
+    public function __construct(
+        private readonly CoasterRepository      $coasterRepository,
```
Comment (florian987) [inline src/Command/CoasterCloseCommand.php:24]: please apply php-cs-fixer rules (config file at root of the project)

**Review by florian987 — CHANGES_REQUESTED** [review]

File: src/Command/CoasterCloseCommand.php (hunk 1 above)
Comment (richardcrete) [reply src/Command/CoasterCloseCommand.php:46]: I think that we can't predict if the coaster will be definitely closed or relocated (usually I don't set a closing date for a temporary closing). We can put a default status for automated closing, the Discord notification will help us to change if needed.

File: src/Command/CoasterCloseCommand.php (hunk 2 above)
Comment (florian987) [reply src/Command/CoasterCloseCommand.php:24]: do you need help?

Comment (richardcrete) [reply src/Command/CoasterCloseCommand.php:24]: I tried to launch the `.php-cs-fixer.dist.php` script (from terminal and Docker container), with `php .php-cs-fixer.dist.php fix src`, but the class PhpCsFixer\Config is not found.

Comment (florian987) [reply src/Command/CoasterCloseCommand.php:24]: I will merge this for now, and explain you how to set up php-cs-fixer later on

**Pull request merged**